src/main/java/org/broadinstitute/dsm/model/ParticipantWrapper.java

```diff
@@ -63,7 +66,7 @@ public JsonObject getDataAsJson() {
                 oncHistoryDetails = OncHistoryDetail.getOncHistoryDetails(instance.getName());
             }
             if (DDPInstance.getRole(instance.getName(), DBConstants.KIT_REQUEST_ACTIVATED)) { //only needed if study is shipping samples per DSM
-                kitRequests = KitRequestShipping.getKitRequests(instance);
+                kitRequests = KitRequestShipping.getKitRequests(instance, ORDER_AND_LIMIT);
```

Comment:
Nice, and it looks like `KitRequestShipping.getKitRequests(@NonNull DDPInstance instance)` is unused so we could remove it!

Reply:
I will keep it for now in case we need it later…